src/include/processor/mapper/plan_mapper.h

```diff
@@ -2,7 +2,7 @@
 
 #include "binder/expression/node_expression.h"
 #include "common/statement_type.h"
-#include "planner/logical_plan/logical_operator/logical_copy.h"
+#include "planner/logical_plan/logical_operator/logical_copy_from.h"
```

Comment:
use planner::LogicalOperator rather than planner::LogicalCopy and move this include to cpp file.